### PR TITLE
update docs landing page

### PR DESCRIPTION
**Pull request by meletj**
Update to the /docs landing page to integrate Accelerate & Pulse better.

### CODE REVIEW

File: src/data/indexData.ts [hunk 1]
```diff
@@ -116,6 +121,34 @@ export const ORMGeneralLinkData = [
     url: '/orm/prisma-client/deployment',
     icon: 'fa-solid fa-book',
   },
+  {
```
Comment (petradonka) [inline src/data/indexData.ts:124]: I think it'd be nice to leave the ORM links on the left, and the new ones to be on the right, so it's aligned with the hero section as well, and makes it easier to skim.

Like this:
CleanShot 2024-05-05 at 11 52 43

File: src/data/indexData.ts [hunk 2]
```diff
@@ -20,70 +20,81 @@ export const ProductLinkData = {
...
+      },
+      {
+        url: '/orm/prisma-migrate',
+        title: 'Prisma Migrate',
+      },
+    ],
+  }, 
+
+  reference: {
+    title: 'Reference',
```
Comment (petradonka) [inline src/data/indexData.ts:66]: Here too, I think it'd be nice to do a left-right logical split to make skimming easier, something like this:
CleanShot 2024-05-05 at 11 55 31

File: src/data/indexData.ts [hunk 3]
```diff
@@ -20,70 +20,81 @@ export const ProductLinkData = {
...
+    ],
+  }, 
+
+  reference: {
+    title: 'Reference', 
+    description: 'Understand the capabilities and usage of our products.',
+    links: [
+      {
+        url: '/orm/reference/prisma-client-reference',
+        title: 'Client API reference',
```
Comment (petradonka) [inline src/data/indexData.ts:71]: Let's either say "reference" for all items, or none of them.

Either:
- "Client API reference" and "Accelerate API reference" or
- "Client API" and "Accelerate API"

File: src/data/indexData.ts [hunk 4]
```diff
@@ -20,70 +20,81 @@ export const ProductLinkData = {
...
+      {
+        url: '/orm/prisma-migrate',
+        title: 'Prisma Migrate',
+      },
+    ],
+  }, 
+
+  reference: {
+    title: 'Reference', 
+    description: 'Understand the capabilities and usage of our products.',
```
Comment (petradonka) [inline src/data/indexData.ts]:
```suggestion
    description: 'Understand the usage and API surface of our products.',
```

I'd consider "capabilities" to not be part of reference docs, which is much more the "dictionary" of our products (error codes, nitty gritty API stuff, etc), vs. more conceptual "capabilities". I think you might've meant "what's possible", and I think API surface covers that in this context.

File: src/data/indexData.ts [hunk 5]
```diff
@@ -116,6 +121,34 @@ export const ORMGeneralLinkData = [
     url: '/orm/prisma-client/deployment',
     icon: 'fa-solid fa-book',
   },
+  {
+    title: `Accelerate`,
+    description:
+      'Set up connection pooling and global cache in your app with Accelerate.',
```
Comment (petradonka) [inline src/data/indexData.ts]:
```suggestion
      'Set up connection pooling and global caching in your app with Accelerate.',
```

File: src/data/indexData.ts [hunk 6]
```diff
@@ -116,6 +121,34 @@ export const ORMGeneralLinkData = [
...
+    title: `Pulse`,
+    description:
+      'React to changes in your database with Pulse.',
+    url: '/pulse/getting-started',
+    icon: 'fa-solid fa-signal-stream',
+  },
+  {
+    title: `Console`,
+    description:
+      'Explore how use our Console to efficiently collaborate with your team on the Prisma Data Platform.',
```
Comment (petradonka) [inline src/data/indexData.ts]:
```suggestion
      'Explore how to use the Platform to easily manage your projects and collaborate within your team.',
```

File: src/data/indexData.ts [hunk 7]
```diff
@@ -116,6 +121,34 @@ export const ORMGeneralLinkData = [
...
+  },
+  {
+    title: `Pulse`,
+    description:
+      'React to changes in your database with Pulse.',
+    url: '/pulse/getting-started',
+    icon: 'fa-solid fa-signal-stream',
+  },
+  {
+    title: `Console`,
```
Comment (petradonka) [inline src/data/indexData.ts]:
```suggestion
    title: `Platform`,
```

File: src/data/indexData.ts [hunk 8]
```diff
@@ -116,6 +121,34 @@ export const ORMGeneralLinkData = [
...
+    description:
+      'Explore how use our Console to efficiently collaborate with your team on the Prisma Data Platform.',
+    url: '/platform/concepts',
+    icon: 'fa-solid fa-chart-simple',
+  },
+  {
+    title: `Platform CLI commands`,
+    description:
+      'Learn how to access the Prisma Data Platform using the integrated commands in the Prisma CLI.',
```
Comment (petradonka) [inline src/data/indexData.ts]:
```suggestion
      'Learn how to access the Platform using the integrated commands in the Prisma CLI.',
```

We haven't shortened PDP to "Platform" a lot in the past, but I think here it'd fit well, also considering that a lot of the docs sections are called Platform.

File: src/pages/index.tsx [hunk 9]
```diff
@@ -135,9 +132,10 @@ function HomepageDatabasesSection() {
         Databases
       </Heading>
       <div className={styles.body}>
-        Prisma ORM works seamlessly across most popular databases and service providers. <br />
+        Prisma ORM works seamlessly across most popular databases and service providers.
         Refer to our <Link to="/orm/reference/database-features">Database features matrix</Link> for information about supported features and types for
-        each database.
+        each database.<br />
+        Explore what databases and database providers <Link to="/accelerate/getting-started#prerequisites">Accelerate</Link> and <Link to="https://www.prisma.io/docs/pulse/database-setup">Pulse</Link> support.
```
Comment (petradonka) [inline src/pages/index.tsx]: This reads more like the inline links will just tell me about Accelerate and Pulse, vs. linking me to the pages they do. Can we rephrase to make that more clear?

**Review by petradonka — CHANGES_REQUESTED** [review]

**Review by mhessdev — APPROVED** [review]

**Review by petradonka — APPROVED** [review]

File: src/pages/index.tsx (hunk 9 above)
Comment (meletj) [reply src/pages/index.tsx]: How about? 

To explore supported databases for Accelerate, visit the [prerequisites section] (/accelerate/getting-started#prerequisites). For Pulse, see the [database setup documentation](/pulse/database-setup).

**Pull request merged**